docs/integrations/sources/file.md

```diff
@@ -66,13 +66,13 @@ File Formats are mostly enabled \(and further tested\) thanks to other open-sour
 | Format | Supported? |
 | :--- | :--- |
 | CSV | Yes |
-| JSON | Experimental |
-| HTML | Untested |
+| JSON | Yes |
```

Comment:
I originally put it as 'Experimental' because Source-File couldn't handle nested JSON files
(flat JSON file worked)